<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>XAttrView</class>
 <widget class="QWidget" name="XAttrView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>606</width>
    <height>710</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="vboxXAttrView" stretch="0,0,1">
   <item>
    <widget class="QGroupBox" name="grpExt2Attributes">
     <property name="title">
      <string>Ext2 Attributes</string>
     </property>
     <layout class="QVBoxLayout" name="vboxExt2Attributes">
      <item>
       <widget class="Ext2AttrView" name="ext2AttrView" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="grpDosAttributes">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>MS-DOS Attributes</string>
     </property>
     <layout class="QVBoxLayout" name="vboxDOSAttributes">
      <item>
       <widget class="DosAttrView" name="dosAttrView" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="grpXAttr">
     <property name="title">
      <string>E&amp;xtended Attributes</string>
     </property>
     <layout class="QVBoxLayout" name="vboxXAttr">
      <item>
       <widget class="QTreeWidget" name="treeXAttr">
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <property name="rootIsDecorated">
         <bool>false</bool>
        </property>
        <property name="uniformRowHeights">
         <bool>true</bool>
        </property>
        <column>
         <property name="text">
          <string>Name</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Value</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Ext2AttrView</class>
   <extends>QWidget</extends>
   <header>xattr/Ext2AttrView.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>DosAttrView</class>
   <extends>QWidget</extends>
   <header>xattr/DosAttrView.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
